drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -502,13 +502,16 @@ def project_configs(filters: nil)
       scope.order(created_at: :desc, id: :asc)
     end
 
+    # Either legacy or coordinated entry referrals can use this query.
+    # Legacy is expected by default (for backwards compatibility); coordinated entry referrals must pass the referral_mode option
     field :project_can_accept_referral, Boolean, 'Whether the destination project is able to accept a referral for the client(s) belonging to the source enrollment', null: false do
       argument :destination_project_id, ID, required: true
       argument :source_enrollment_id, ID, required: true
+      argument :referral_mode, type: HmisSchema::Enums::ReferralMode, required: false, default_value: 'legacy'
     end
-    def project_can_accept_referral(destination_project_id:, source_enrollment_id:)
+    def project_can_accept_referral(destination_project_id:, source_enrollment_id:, referral_mode:)
       source_enrollment = Hmis::Hud::Enrollment.viewable_by(current_user).find(source_enrollment_id)
-      raise 'access denied' unless current_permission?(permission: :can_manage_outgoing_referrals, entity: source_enrollment.project)
+      access_denied! unless current_permission?(permission: :can_manage_outgoing_referrals, entity: source_enrollment.project)
```

Comment:
perhaps use a policy check here? Would have to define `can_send_ce_referral?` if it's helpful to have distinct permissions
```suggestion
      policy = policy_for(source_enrollment.project, policy_type: :hmis_project)
      can_send = referral_mode == 'legacy' ? policy.can_send_direct_referral? : policy.can_send_ce_referral?
      access_denied! unless can_send
```

Reply:
Agreed I think we can use the same policy  method martha added elsewhere in this PR: `policy_for(source_enrollment.project, policy_type: :hmis_project).can_send_out_direct_referral?`

Reply:
For sure, we should re-use the direct referral check. I guess the intent is to check both types of referrals here - So we probably don't want to check the direct referral perm unless referral_mode == 'legacy'? Something like:
```ruby
 can_send = referral_mode == 'legacy' ? policy.can_send_out_direct_referral? : true
 access_denied! unless can_send
```

Reply:
The same permission is used to govern the sending of legacy and "ce" direct referrals, so they would both be doing the same underlying check